qcodes/instrument_drivers/Keysight/private/Keysight_344xxA_submodules.py

```diff
@@ -756,6 +767,17 @@ def _licenses(self) -> Sequence[str]:
             return licenses_list
         return tuple()
 
+    def _options(self) -> Sequence[str]:
```

Comment:
Please don't use Abstract types in return position. We know this is a tuple so the correct type is `Tuple[str, ...]`

Reply:
done. good catch!